lib/diagnostic.rb

```diff
@@ -105,15 +119,21 @@ def eat_meat(food)
 end
 
 ##
-# your response here
+class Lion < Cat
+  include Carnivorus
+  def roar
+    print "ROAR!"
+  end
+end
 ##
 
 # #
 # What are some of the advantages of using composition (i.e. mixins)
 # over using direct inheritance?
 
 ##
-# your response as a comment here
+# Mixins are preventing us form repeating our selves that resulst with D.R.Y
```

Comment:
Composition allows a class to draw from multiple sources whereas inheritance only allows a class to draw from one source.